manifests/unit_file.pp

```diff
@@ -57,4 +65,14 @@
     mode    => '0444',
     notify  => Class['systemd::systemctl::daemon_reload'],
   }
+
+  if $enable != undef or $active != undef {
```

Comment:
Previously it was mentioned there are other unit files like ```mount``` or ```timer```. Can ```service``` handle those as well or should we use a regex like ```$name =~ /^.+\.service$/``` in this if as well?

Reply:
The `service` native resource can definitely handle other types of unit files, as long as the provider is set as `systemd`.

Reply:
It should be noted, however, that unless a unit type suffix is used as the resource name (eg. `.mount` or `.timer`, it will default to a service unit type.